### PR TITLE
STunnel: Tunnel list does not show certificate

### PR DESCRIPTION
**Pull request by vktg**
Redmine Issue: https://redmine.pfsense.org/issues/9220
 Ready for review

save certificate name in certname field

### CODE REVIEW

File: security/pfSense-pkg-stunnel/Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 PORTNAME=	pfSense-pkg-stunnel
 PORTVERSION=	5.50
-PORTREVISION=	2
+PORTREVISION=	3
```
Comment (rbgarga) [inline security/pfSense-pkg-stunnel/Makefile]: #718 already bumped it to 3 here, please rebase your fork and then bump to 4

File: security/pfSense-pkg-stunnel/files/usr/local/pkg/stunnel.inc [hunk 2]
```diff
@@ -77,6 +78,7 @@ function stunnel_save() {
 		fwrite($fout, base64_decode($pkgconfig['custom_options']) . "\n");
 	}
 	fclose($fout);
+	write_config();
```
Comment (rbgarga) [inline security/pfSense-pkg-stunnel/files/usr/local/pkg/stunnel.inc]: Please add a description to write_config()

Comment (vktg) [reply security/pfSense-pkg-stunnel/files/usr/local/pkg/stunnel.inc]: done

File: security/pfSense-pkg-stunnel/Makefile (hunk 1 above)
Comment (vktg) [reply security/pfSense-pkg-stunnel/Makefile]: done

**Pull request merged**